### PR TITLE
edit readme again...

### PR DESCRIPTION
**Pull request by ayhansipahi**

### CODE REVIEW

**Comment by peoray** [comment]
What exactly did you do?

**Comment by ayhansipahi** [comment]
this is the first pull request 

1. I opened a branch named `feature/test`
2. Do changes and commit
3. push to remote
4. go to github and check branches and create pull request from my branch
5. thats it

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Geckos-Team-33
 
-
```
Comment (ayhansipahi) [inline README.md:3]: we can review out commits and changes we made

**Review by ayhansipahi — COMMENTED** [review]
a review

**Comment by peoray** [comment]
So basically, you were just testing?

**Pull request merged**